5.14.0/Dockerfile

```diff
@@ -6,11 +6,8 @@ ENV ACTIVEMQ_TCP=61616 ACTIVEMQ_AMQP=5672 ACTIVEMQ_STOMP=61613 ACTIVEMQ_MQTT=188
 
 ENV ACTIVEMQ_HOME /opt/activemq
 
-RUN \
-    curl -O http://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz && \
-    mkdir -p /opt && \
-    tar xf $ACTIVEMQ-bin.tar.gz -C /opt/ && \
-    rm $ACTIVEMQ-bin.tar.gz && \
+RUN set -x && \
+    curl -s http://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /opt && \
```

Comment:
I like this line. Could you add -S to the curl command line so that it looks like `curl -sS ...` so that we see errors when curl fails?